src/__tests__/pages/api/youtube/continuation.test.ts

```
@@ -0,0 +1,281 @@
+      body: buildRequestBody({ apiKey: '' }),
+    })
+
+    await handler(req as any, res as any)
+
+    expect(res._getStatusCode()).toBe(400)
+    expect(JSON.parse(res._getData())).toEqual({
+      error: 'Empty API Key',
+      errorCode: 'EmptyAPIKey',
+    })
+
+    Object.keys(process.env).forEach((key) => delete process.env[key])
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)